coderadar-rest/src/test/java/io/reflectoring/coderadar/rest/unit/useradministration/teams/ListTeamsForUserControllerTest.java

```diff
@@ -0,0 +1,68 @@
+package io.reflectoring.coderadar.rest.unit.useradministration.teams;
+
+import static org.mockito.Mockito.mock;
+
+import io.reflectoring.coderadar.rest.domain.GetTeamResponse;
+import io.reflectoring.coderadar.rest.unit.UnitTestTemplate;
+import io.reflectoring.coderadar.rest.useradministration.teams.ListTeamsForUserController;
+import io.reflectoring.coderadar.useradministration.UserNotFoundException;
+import io.reflectoring.coderadar.useradministration.domain.Team;
+import io.reflectoring.coderadar.useradministration.domain.User;
+import io.reflectoring.coderadar.useradministration.port.driver.teams.get.ListTeamsForUserUseCase;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+class ListTeamsForUserControllerTest extends UnitTestTemplate {
+
+  private final ListTeamsForUserUseCase listTeamsForUserUseCase =
+      mock(ListTeamsForUserUseCase.class);
+  private final ListTeamsForUserController testController =
+      new ListTeamsForUserController(listTeamsForUserUseCase);
+
+  @BeforeEach
+  void setUp() {
+    Mockito.when(listTeamsForUserUseCase.listTeamsForUser(5L))
```

Comment:
Ist der TestCase so angelegt, dass der User der die Methode aufruft, in beiden Teams drin ist und zusätzlich in TestTeam2 noch TestUser1 ist? Eigentlich sollte ein Nutzer, Nutzer doch auch entsprechend in dem Team sein oder?

Reply:
Hier ist der einzige Nutzer TestUser1, dieser sollte aber die ID 5 haben und in beide Teams drin sein.